app/policies/conversation_policy.rb

```diff
@@ -0,0 +1,11 @@
+class ConversationPolicy < ApplicationPolicy
+  def index?
+    user_authenticated?
+  end
+
+  private
+
+  def user_authenticated?
```

Comment:
I see you have this same method in `target_policy`. Maybe you can add it as a private method into `ApplicationPolicy ` so it can be used here too in
```
def index?
    user_authenticated?
  end
```
then you can delete this private method from `ConversationPolicy` and `TargetPolicy` , WDYT?

Reply:
Done! Yep make sens, now i call directly user authenticated in Conversation Policy and Target Policy

Reply:
Done!